backend/src/handlers/dashboard.go

```diff
@@ -97,40 +97,44 @@ func (srv *Server) handleLoginMetrics(w http.ResponseWriter, r *http.Request, lo
 		if err != nil {
 			return newDatabaseServiceError(err)
 		}
-		totalUsers, err := srv.Db.GetTotalUsers(facilityId)
+		totalStudents, totalAdmins, err := srv.Db.GetTotalUsers(facilityId)
```

Comment:
This is going to be challenging since we're changing vernacular but we should probably use `totalResidents` instead of `totalStudents`.